change/@itwin-imodel-transformer-af619951-5fbf-47a8-a92c-7c1027ff2cb1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
we're introducing a new IModelExportHandler API, can we make this a minor version bump?